chore(docs): Kubeconfig breaking change post 1.34 (#419)
spinnaker/spinnaker.io: content/en/docs/releases/next-release-preview/index.md

https://github.com/spring-cloud/spring-cloud-release/wiki/Supported-Versions#supported-releases

In order to use compatible Spring Cloud version along with Spring Boot, upgrade Spring Cloud from 2020.0.6 used in Spinnaker 1.33.0 to 2021.0.8 as part of Spinnaker 1.34.0. 

### UPCOMING BREAKING CHANGE:

> kubectl in the latest releases have removed support for external auth flows using the `apiVersion: client.authentication.k8s.io/v1alpha1` exec API.  1.34 will include an aws cli (version 1.24) which supports both old and new exec api versions.  Additionally the aws-iam-authenticator binary also supports both exec api versions.   1.34 is the last release that will support v1alpha1 exec APIs in kubeconfig files.  Please update your kubeconfig files to use the v1beta1 api version.  Please see the upcoming [PR that will remove suppport for these after this release.](https://github.com/spinnaker/clouddriver/pull/6156)

Reviewer: ```suggestion
kubectl in the latest releases has removed support for external auth flows using the `apiVersion: client.authentication.k8s.io/v1alpha1` exec API.  1.34 includes an aws cli (version 1.24) which supports both old and new exec api versions.  Additionally the aws-iam-authenticator binary also supports both exec api versions.   1.34 is the last release that will support v1alpha1 exec APIs in kubeconfig files.  Please update your kubeconfig files to use the v1beta1 api version.  Please see the upcoming [PR that will remove support for these after this release.](https://github.com/spinnaker/clouddriver/pull/6156)
```
